### PR TITLE
Bioexcel summer school

### LINKED ISSUE #675: Update tutorial for BioExcel summer school 2024
Make use of new haddock3 functionalities and webapp

### PR DESCRIPTION
**Pull request by VGPReys**
Closes #675 .

- [x] updated configuration files
- [x] updated surfdrive link
- [x] webapp zip archive to be loaded (surfdrive)
- [x] updated performance values to match current pre-runs
- [x] new bonus 5 related to the use of the webapp
- [ ] some fixes must be applied on the webapp side to fit the needs

### CODE REVIEW

**Review by amjjbonvin — APPROVED** [review]
At this time is looks like an exact duplication of the 2023 tutorial, correct?

**Comment by VGPReys** [comment]
> At this time is looks like an exact duplication of the 2023 tutorial, correct?

No, as stated in the PR header
- surfdrive link was modified
- analysis values were updated
- configuration file now used `[seletop]  select = 50`
- new parameter added in `[clustfcc]  plot_matrix = true`
- Some typo were modified
- bonus 5 was added.

**Comment by amjjbonvin** [comment]
All good!

File: education/HADDOCK3/index.md
Comment (AnnaKravchenko) [inline education/HADDOCK3/index.md:1]: add a short description of this tutorial ?

Comment (amjjbonvin) [reply education/HADDOCK3/index.md:1]: It is an update of the current BioExcel 2023 one. We will not have a new entry for it (if I am correct)

**Comment by amjjbonvin** [comment]
Then indeed a short description is needed. 

And we can probably remove then the 2023 version since it will be redundant

And we rename it to remove the year

File: education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md [hunk 1]
```diff
@@ -0,0 +1,2530 @@
...
+**Note** that with the current implementation, the first user to register comes the administrator.
+
+</details>
+
+
+### Running this tutorial
+
+Once logged in, click on the `build` menu to start the creation of a custom workflow.
+You will land in the workflow-builder page, where you can interactively build your haddock3 scenario by combining the available modules.
+This page is sub-divided in three areas, that will be described below.
```
Comment (AnnaKravchenko) [inline education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md:2335]: Add screenshot to complement the text description

Comment (AnnaKravchenko) [reply education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md:2335]: This page is sub-divided in three areas, that will be described below. >>  This page is subdivided into three areas described below.

File: education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md [hunk 2]
```diff
@@ -0,0 +1,2530 @@
...
+
+On the left are presented the list of modules.
```
Comment (AnnaKravchenko) [inline education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md:2337]: On the left are presented the list of modules. >> On the left is presented the list of modules.

File: education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md [hunk 3]
```diff
@@ -0,0 +1,2530 @@
...
+To add a module to the workflow, just click on it, and it will be automatically added at the bottom of the configuration file.
+Alternatively, you can drag-and-drop it to the central panel, at the location where you wish to place it.
+
+In the central panel is presented the current set of modules defining your workflow.
```
Comment (AnnaKravchenko) [inline education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md:2341]: In the central panel is presented the current set of modules defining your workflow. >> The set of modules defining your current workflow is presented on the central panel.

File: education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md [hunk 4]
```diff
@@ -0,0 +1,2530 @@
...
+</details>
+
+<br>
+
+**Note** that you can also upload a zip archive of a workflow containing a configuration file named `workflow.cfg` and all corresponding files (e.g: pdb structures, restraints files, topological parameters, ...). Workflow archives presented in this tutorial are available in `workflows/webapp-workflows/`.
+
+
+### Loading haddock3 runs
+
+The web application also allows one to upload a pre-computed run, allowing to enjoy the graphical interface to navigate thought the docking results.
```
Comment (AnnaKravchenko) [inline education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md:2395]: The web application also allows one to upload a pre-computed run, allowing to enjoy the graphical interface to navigate thought the docking results. >> The webapp also allows you to upload a pre-computed run, so you can navigate through the docking results with ease thanks to the graphical interface.

(fits better in the text style)

File: education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md [hunk 5]
```diff
@@ -0,0 +1,2530 @@
...
+Under the `Upload` menu, one can upload two types of zip archives:
```
Comment (AnnaKravchenko) [inline education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md:2396]: one can upload >>  you can upload

File: education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md [hunk 6]
```diff
@@ -0,0 +1,2530 @@
...
+</details>
+
+<br>
+
+**Note** that the archives of workflows are available in `workflows/webapp-workflows/`, and archives of pre-computed runs are stored in `runs/webapp_runs/`.
+
+
+### Navigating throught the results
+
+In the `Manage` page, a table is displaying all the haddock3 runs performed by one user.
```
Comment (AnnaKravchenko) [inline education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md:2421]: In the `Manage` page, a table is displaying all the haddock3 runs performed by one user. >> On the `Manage` page, a table displays all the haddock3 runs performed by one user.

File: education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md [hunk 7]
```diff
@@ -0,0 +1,2530 @@
...
+This table contains the job status (queued, running, error, ok), its name, creation date and modification date.
+On the right side of the table, actions can be performed.
+Current implementation allows to rename a run or to delete it.
+
+To access the content of a run, click on its name and you will be directed to the haddock3 webapp result page.
+You will land on the analysis page summarizing performances of the complexes obtained at the last stage, similar to what was previously obtained by opening the `report.html` file (see above) that contains a variety of plots displaying the HADDOCK score and its components against various CAPRI metrics.
```
Comment (AnnaKravchenko) [inline education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md:2427]: You will land on the analysis page, which summarizes the performance of the models obtained at the last stage. This is similar to the previous method of opening the `report.html` file (see above), which contains various plots displaying the HADDOCK score and its components against different CAPRI metrics.

File: education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md [hunk 8]
```diff
@@ -0,0 +1,2530 @@
...
+In this case, because a reference was provided during the `caprieval` module, performances are evaluated based on this structure.
+
+In addition, you can click on the `browse` button, that will let you access all the files of the run.
```
Comment (AnnaKravchenko) [inline education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md:2430]: button, that will >>  button, which will

File: education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md [hunk 9]
```diff
@@ -0,0 +1,2530 @@
...
+
+# ====================================================================
+
+{% endhighlight %}
+
+
+To simplify the tutorial, scoring scenario configuration files are provided in the `workflow/` directory, precomputated results in the `runs/` directory and finally archives for the haddock3-webapp upload section in `workflow/webapp/scoring-*.cfg`.
+
+<a class="prompt prompt-question">
+How are scoring the AlphaFold predictions ?
```
Comment (AnnaKravchenko) [inline education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md:2497]: How are scoring the AlphaFold predictions ? >> How are scoring the AlphaFold predictions?

File: education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md [hunk 10]
```diff
@@ -0,0 +1,2530 @@
...
+</a>
+
+<a class="prompt prompt-question">
+Is the HADDOCK scoring function able to detect best models ?
```
Comment (AnnaKravchenko) [inline education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md:2501]: Is the HADDOCK scoring function able to detect best models ? >> Can the HADDOCK scoring function identify the best models?

Comment (AnnaKravchenko) [reply education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md:2501]: Consider adding an answer to this question

File: education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md (hunk 9 above)
Comment (AnnaKravchenko) [reply education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md:2497]: Maybe rephrase to smth like "How accurate is the scoring of the AlphaFold predictions?”

**Review by AnnaKravchenko — APPROVED** [review]
Looking good!
I spot some typos and suggested several sentences that can benefit if rephrased.

File: education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md [hunk 11]
```diff
@@ -0,0 +1,2530 @@
...
+
+The web application also allows one to upload a pre-computed run, allowing to enjoy the graphical interface to navigate thought the docking results.
+Under the `Upload` menu, one can upload two types of zip archives:
+
+* **Workflow**: a zip archive containing a configuration file named `workflow.cfg` and all corresponding files (e.g: pdb structures, restraints files, topological parameters, ...)
+* **Run**: a zip archive of the run.
+
+<details style="background-color:#DAE4E7">
+ <summary>
+  <b><i>How to generate a zip archive a the run? </i></b> <i class="material-icons">expand_more</i>
```
[the remark this answers is not in the document]
Comment (VGPReys) [reply education/HADDOCK3/HADDOCK3-antibody-antigen-bioexcel2024/index.md:2415]: >>> How to generate a zip archive of a run?
